src/components/Blocks/Banner/Banner.tsx

```diff
@@ -52,6 +57,15 @@ export const Banner = ({
 }: BannerProps) => {
   const platform = usePlatform();
   const hasBackground = hasReactNode(background);
+  let descriptionNode;
+
+  if (hasReactNode(description)) {
```

Comment:
Let's do it like in the rest of the library, for example `src/components/Blocks/Button/components/ButtonTypography/ButtonTypography.tsx`. You would need to create `Block/Banner/components/BannerDescriptionTypography/BannerDescriptionTypography.tsx`